src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,19 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        User result = new User();
+        for (int i = 0; i < users.length; i++) {
+            if (email.equals(users[i].getEmail())) {
+                result = users[i];
+                break;
+            } else {
+                result = null;
+            }
+        }
+        return result;
+    }
+
+    public User[] getUsers() {
+        return users;
```

Comment:
so difficult to read
```suggestion
    public User findByEmail(String email) {
        for (User user : users) {
            if (user.getEmail().equals(email)) {
                return user;
            }
        }
        return null;
    }
```